data/sql/dbfix/2016_08_23_00.sql

```diff
@@ -0,0 +1,144 @@
+(@PATH,6,324.4433,-6101.173,115.008,0,0,1,0,100,0),
+(@PATH,7,351.7174,-6115.625,115.008,0,0,1,0,100,0),
+(@PATH,8,380.059,-6146.51,102.4525,0,0,1,0,100,0),
+(@PATH,9,381.592,-6162.233,80.36916,0,0,1,0,100,0),
+(@PATH,10,363.2276,-6145.913,80.36916,0,0,1,0,100,0),
+(@PATH,11,338.3981,-6105.44,80.36916,0,0,1,0,100,0),
+(@PATH,12,319.2164,-6096.714,84.36917,0,0,1,0,100,0),
+(@PATH,13,290.8945,-6070.151,84.36917,0,0,1,0,100,0);
+
+-- Pathing for Entry: 24787 'TDB FORMAT' 
+SET @NPC := 115845;
+SET @PATH := @NPC * 10;
+UPDATE `creature` SET `spawndist`=0,`MovementType`=2,`position_x`=360.9156,`position_y`=-6061.682,`position_z`=302.8403 WHERE `guid`=@NPC;
+DELETE FROM `creature_addon` WHERE `guid`=@NPC;
+INSERT INTO `creature_addon` (`guid`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES (@NPC,@PATH,0,0,1,0, '43775');
+DELETE FROM `waypoint_data` WHERE `id`=@PATH;
+INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
+(@PATH,1,360.9156,-6061.682,302.8403,0,0,1,0,100,0),
+(@PATH,2,389.247,-6078.495,302.8403,0,0,1,0,100,0),
+(@PATH,3,392.2037,-6094.839,296.007,0,0,1,0,100,0),
+(@PATH,4,374.589,-6096.534,288.5903,0,0,1,0,100,0),
+(@PATH,5,358.6935,-6082.393,274.9237,0,0,1,0,100,0),
+(@PATH,6,338.8949,-6049.761,274.9237,0,0,1,0,100,0),
+(@PATH,7,320.3495,-6021.407,274.9237,0,0,1,0,100,0),
+(@PATH,8,306.0276,-5998.132,285.257,0,0,1,0,100,0),
+(@PATH,9,313.4884,-5983.751,285.2291,0,0,1,0,100,0),
+(@PATH,10,320.7385,-5972.151,286.1277,0,0,1,0,100,0),
+(@PATH,11,330.6828,-6007.996,302.8403,0,0,1,0,100,0),
+(@PATH,12,349.4529,-6033.62,302.8403,0,0,1,0,100,0),
+(@PATH,13,360.9156,-6061.682,302.8403,0,0,1,0,100,0);
+
+-- Pathing for Entry: 24787 'TDB FORMAT' 
+SET @NPC := 115844;
+SET @PATH := @NPC * 10;
+UPDATE `creature` SET `spawndist`=0,`MovementType`=2,`position_x`=389.949,`position_y`=-6113.926,`position_z`=217.1557 WHERE `guid`=@NPC;
+DELETE FROM `creature_addon` WHERE `guid`=@NPC;
+INSERT INTO `creature_addon` (`guid`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES (@NPC,@PATH,0,0,1,0, '43775');
+DELETE FROM `waypoint_data` WHERE `id`=@PATH;
+INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
+(@PATH,1,389.949,-6113.926,217.1557,0,0,1,0,100,0),
+(@PATH,2,365.9727,-6083.048,249.1279,0,0,1,0,100,0),
+(@PATH,3,348.9503,-6058.797,249.1279,0,0,1,0,100,0),
+(@PATH,4,334.4755,-6041.239,249.1279,0,0,1,0,100,0),
+(@PATH,5,306.4679,-6010.077,240.8223,0,0,1,0,100,0),
+(@PATH,6,305.4742,-6011.148,217.1557,0,0,1,0,100,0),
+(@PATH,7,308.0244,-6019.713,217.1557,0,0,1,0,100,0),
+(@PATH,8,331.7084,-6052.974,217.1557,0,0,1,0,100,0),
+(@PATH,9,350.9348,-6071.563,217.1557,0,0,1,0,100,0),
+(@PATH,10,371.7133,-6102.678,217.1557,0,0,1,0,100,0),
+(@PATH,11,389.949,-6113.926,217.1557,0,0,1,0,100,0);
```

Comment:
this file should go under [data/sql/updates/pending_db_world](https://github.com/azerothcore/azerothcore-wotlk/tree/master/data/sql/updates/pending_db_world)